src/containers/AsideNavigation/useNavigationMenuItems.tsx

```diff
@@ -45,7 +46,7 @@ export function useNavigationMenuItems() {
                 title: i18n('pages.query'),
                 icon: Terminal,
                 iconSize: 20,
```

Comment:
Please remove this property, it is set for all items in a row in `return`.

Reply:
That's the original code, I made no changes xd

Reply:
I agree it should be done this way btw